WordPress/Sniffs/VIP/ValidatedSanitizedInputSniff.php

```diff
@@ -45,6 +45,26 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr )
 	{
 		$tokens = $phpcsFile->getTokens();
 
+		/*
+		 * Quoting 'http://php.net/manual/en/reserved.variables.server.php' :
+		 *
+		 * There is no guarantee that every web server will provide any of these;
+		 * servers may omit some, or provide others not listed here.
+		 *
+		 * That said, a large number of these variables are accounted for in
+		 * the » CGI/1.1 specification, so you should be able to expect those.
+		 * http://www.faqs.org/rfcs/rfc3875.html
+		 *
+		 * @see https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/191
+		 */
+		$server_vars = array(
+			'AUTH_TYPE', 'CONTENT_LENGTH', 'CONTENT_TYPE', 'GATEWAY_INTERFACE',
+			'PATH_INFO', 'PATH_TRANSLATED', 'QUERY_STRING', 'REMOTE_ADDR',
+			'REMOTE_HOST', 'REMOTE_IDENT', 'REMOTE_USER', 'REQUEST_METHOD',
+			'SCRIPT_NAME', 'SERVER_NAME', 'SERVER_PORT', 'SERVER_PROTOCOL',
+			'SERVER_SOFTWARE',
+		);
```

Comment:
Related: [#34394](https://core.trac.wordpress.org/ticket/34394) – Initialise `$_SERVER['SERVER_NAME']` in the unit test bootstrap